.codespellignore

```diff
@@ -0,0 +1,8 @@
+altho
+datas
+edn
+juxt
+methodd
+pass-thru
```

Comment:
This seems to only exist in comments, so it should be inconsequential to correct. I see the referenced issue in this comment variously uses "pass thru" and "pass through" interchangeably.